### PR TITLE
Add support for small button variant

### PR DESCRIPTION
**Pull request by julbar**

### CODE REVIEW

File: warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt [hunk 1]
```diff
@@ -172,13 +183,18 @@ fun WarpButton(
...
         val colors = if (loading) loadingColors else buttonColors
         //disable click when button is in loading state
         val clickAction = if (loading) {
             {}
         } else onClick
 
+        val paddingVertical = if (small) dimensions.space025 else dimensions.space2
+        val paddingHorizontal = if (small) dimensions.space025 else dimensions.space2
+        val loadingModifier = if (loading) modifier.loadingAnimation() else modifier
+        val smallModifier =
```
Comment (shining-cat) [inline warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt:195]: you could go with the `.then` conditional attritbute for Modifiers: https://medium.com/@sujathamudadla1213/module-4-custom-modifiers-advanced-topics-topic-modifier-then-174414608bcd

File: warp/src/main/java/com/schibsted/nmp/warp/theme/WarpDimensions.kt [hunk 2]
```diff
@@ -41,6 +41,7 @@ object WarpDimensions {
 object WarpComponentDimensions {
     val stepIndicator = StepIndicatorDimensions
     val pillIcon = 16.dp
+    val smallButtonHeight = 35.dp
```
Comment (shining-cat) [inline warp/src/main/java/com/schibsted/nmp/warp/theme/WarpDimensions.kt:44]: shouldn't these dimensions be tied to the warp dimensions rather?

Comment (julbar) [reply warp/src/main/java/com/schibsted/nmp/warp/theme/WarpDimensions.kt:44]: These are custom

File: warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt (hunk 1 above)
Comment (julbar) [reply warp/src/main/java/com/schibsted/nmp/warp/components/WarpButton.kt:195]: I have used that :) Line 197

**Review by martintoften — APPROVED** [review]
👍

**Pull request merged**